scripts/x-date-pickers.exports.json

```diff
@@ -111,7 +111,6 @@
   { "name": "FieldSectionContentType", "kind": "TypeAlias" },
   { "name": "FieldSectionType", "kind": "TypeAlias" },
   { "name": "FieldSelectedSections", "kind": "TypeAlias" },
-  { "name": "FieldValueType", "kind": "TypeAlias" },
```

Comment:
We should export the new type and treat that as a breaking change.

Reply:
Agreed.
WDYT about `PickerValueType`?
Would that be too confusing beside `PickerValue`? 🤔

Reply:
On another note, do we have a clear use-case for why we are exporting this type? 🤔